docs/fundamentals/devcontainers.md

```diff
@@ -0,0 +1,15 @@
+---
+title: .NET Aspire and Devcontainers in Visual Studio Code
```

Comment:
We have `.NET Aspire` automatically being appended to all docs titles, for SEO, we can omit it from this part so it's not redundant.

```suggestion
title: Devcontainers in Visual Studio Code
```